jupyter-extension/src/plugins/mount/components/SortableList/SortableList.tsx

```diff
@@ -10,20 +10,37 @@ type SortableListProps = {
   open: (path: string) => void;
   updateData: (data: ListMountsResponse) => void;
   mountedItems: Mount[];
+  type: string;
 };
 
 const nameComparator = {
   name: 'Name',
   func: stringComparator,
-  accessor: (item: Mount | Repo) => ('name' in item ? item.name : item.repo),
+  accessor: (item: Mount | Repo) =>
+    'name' in item ? item.name : `${item.project}_${item.repo}`,
 };
 
 const SortableList: React.FC<SortableListProps> = ({
   open,
   items,
   updateData,
   mountedItems,
+  type,
 }) => {
+  const allProjectsLabel = 'All projects';
+  // .map() likes Array<Mount | Repo> but not Mount[] | Repo[]
+  // https://github.com/microsoft/TypeScript/issues/33591#issuecomment-841832390
+  const _items: Array<Mount | Repo> = items;
```

Comment:
You might be able to get rid of this by moving this type to the prop instead.

```tsx
type SortableListProps = {
  items: Array<Mount | Repo>;
```
or

```tsx
type SortableListProps = {
  items: (Mount | Repo)[];
```

Alternatively using an interface instead of a type for the props might solve it but if it works it works